spring-security/src/main/java/com/sap/cloud/security/spring/token/authentication/XsuaaTokenAuthorizationConverter.java

```diff
@@ -41,14 +41,18 @@ public AbstractAuthenticationToken convert(Jwt jwt) {
 	}
 
 	protected Collection<GrantedAuthority> localScopeAuthorities(Jwt jwt) {
-		Collection<GrantedAuthority> localScopeAuthorities = new ArrayList<>();
 		Collection<String> scopes = jwt.getClaimAsStringList(TokenClaims.XSUAA.SCOPES);
 		if (scopes == null) {
 			return Collections.emptySet();
 		}
+		return localScopeAuthorities(jwt, scopes);
+	}
+
+	protected Collection<GrantedAuthority> localScopeAuthorities(Jwt jwt, Collection<String> scopes) {
```

Comment:
the jwt seems to be not used in the method, is it required to have it in the argument list?

Reply:
good finding! 
But in this case this protected method is foreseen as "enhancement spot" and app developers may need access to other token claims.